src/components/Toast/Toast.tsx

```diff
@@ -74,10 +74,11 @@ const StyledIcon = styled(Icon)`
 
 const stopPropagation = (event) => event?.stopPropagation();
 
-const Toast: React.FC<ToastProps> = ({ onClose, children }) => (
+const Toast: React.FC<ToastProps> = ({ onClose, children, width }) => (
   <ToastContainer
     alignItems="flex-start"
     justifyContent="space-between"
+    width={width}
```

Comment:
For internal props passed into styled-components please use `$...` prefix. We want to prevent propagation of props into DOM so we are using SC's [Transient Props](https://styled-components.com/docs/api#transient-props)
```suggestion
    $width={width}
```